package.json

```diff
@@ -23,6 +23,8 @@
     "classnames": "^2.2.6",
     "clipboard-copy": "^3.0.0",
     "date-fns": "2.0.0-alpha.37",
+    "dotenv": "^8.0.0",
+    "dotenv-cli": "^2.0.0",
```

Comment:
These are not required in the codebase so we could make them dev dependencies.

Reply:
Ah, of course. Let's do that.